package.json

```diff
@@ -126,7 +126,7 @@
     "tracking-protection": "1.1.x",
     "underscore": "1.8.3",
     "url-loader": "^0.5.7",
-    "webtorrent-remote": "^2.0.0"
+    "webtorrent-remote": "^2.0.2"
```

Comment:
it may be a good idea to submit webtorrent-remote <2.0.2 to the Node Security Platform: https://nodesecurity.io/report. That way, `nsp check` will fail in this repo for dev users who have a lesser version and they will know to upgrade.